.devcontainer/devcontainer.json

```diff
@@ -1,10 +1,10 @@
 {
     "name": "Development Jellyfin Server",
-    "image":"mcr.microsoft.com/devcontainers/dotnet:8.0-jammy",
+    "image":"mcr.microsoft.com/devcontainers/dotnet:8.0-bookworm",
     // restores nuget packages, installs the dotnet workloads and installs the dev https certificate
-    "postStartCommand": "dotnet restore; dotnet workload update; dotnet dev-certs https --trust",
+    "postStartCommand": "sudo dotnet restore; sudo dotnet workload update; sudo dotnet dev-certs https --trust; sudo bash \"./.devcontainer/install-ffmpeg.sh\"",
```

Comment:
Why does this need sudo now?

Reply:
It just seems like that when you run it locally, its required:

```

[30001 ms] Start: Run in container: /bin/sh -c dotnet restore; dotnet workload update; dotnet dev-certs https --trust
An issue was encountered verifying workloads. For more information, run "dotnet workload update".
  Determining projects to restore...
  Restored /workspaces/jellyfin/Jellyfin.Data/Jellyfin.Data.csproj (in 3.14 sec).
...
  Restored /workspaces/jellyfin/Jellyfin.Server.Implementations/Jellyfin.Server.Implementations.csproj (in 6.22 sec).
Inadequate permissions. Run the command with elevated privileges.
Trusting the HTTPS development certificate was requested. Trust is per-user and may require additional configuration. See https://aka.ms/dev-certs-trust for more information.
Successfully created and trusted a new HTTPS certificate.
Running the postAttachCommand from devcontainer.json...
```

Otherwise it says it cannoot update because that seems to require sudo. So to be sure then we can run everything

Reply:
Nope. Its checked out on the root system and the commands run in the container itself even when not sudoing as the vscode user for example.